### PR TITLE
Add symbol to IMMUTABLE_TYPES. Fixes #30

### LINKED ISSUE #30: Incompatible with React
When using something like 

``` JavaScript
const Component = () => (
  <div>
    {
       immuArray.map(() => <div>test</test>)  
    }
  </div>
)
```

React throws:

```
Uncaught Invariant Violation: Objects are not valid as a React child (found: object with keys {$$typeof, type, key, ref, props, _owner, _store}). If you meant to render a collection of children, use an array instead or wrap the object using createFragment(object) from the React add-ons. 
```

react: 0.14.8,
immu: 2.0.1

### PR DESCRIPTION
**Pull request by ghost**
Fixes #30

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,7 +3,8 @@ const IMMUTABLE_TYPES = {
   string: true,
   boolean: true,
   number: true,
-  undefined: true
+  undefined: true,
+  symbol: true
```
Comment (scottcorgan) [inline src/index.js:7]: I haven't done JS in quite awhile. Are Symbols immutable?

File: __test__/index.js [hunk 2]
```diff
@@ -31,6 +31,10 @@ test('returns fast for non-object values', ({equal}) => {
   let immuBool = immu(bool)
   equal(immuBool, bool, 'does nothing to boolean')
 
+  let sym = Symbol('sym')
+  let immuSym = immu(sym)
+  equal(immuSym, sym, 'does nothing to boolean')
```
Comment (scottcorgan) [inline __test__/index.js:36]: 👍 Thanks for adding tests

File: src/index.js (hunk 1 above)
Comment (ghost) [reply src/index.js:7]: yes

**Comment by scottcorgan** [comment]
Cool. Released as `3.0.0`

**Pull request merged**